### ISSUE TITLE
InitialHandler - bad packet ID

### ISSUE THREAD

**Issue by DanielAlves123** [SpigotMC/BungeeCord#2806]
My console is spammed in these errors, probably some ddos attack on proxy
[WARNING] [/x:x] <-> InitialHandler - bad packet ID, are mods in use!? Did not read all bytes from packet class net.md_5.bungee.protocol.packet.Handshake 0 Protocol HANDSHAKE Direction TO_SERVER

**Comment by Black-Hole** [comment]
This is no BungeeCord issue.
Please use the Spigot forums for community support for help mitigating such attacks.

**Comment by DanielAlves123** [comment]
@Black-Hole it happened on bungee.

**Comment by Black-Hole** [comment]
Yes, but the issue is the DDoS attack, not those otherwise useful messages.
